opencraft/tests/test_views.py

```diff
@@ -1,109 +0,0 @@
-# -*- coding: utf-8 -*-
-#
-# OpenCraft -- tools to aid developing and hosting free software projects
-# Copyright (C) 2015-2019 OpenCraft <contact@opencraft.com>
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero General Public License as
-# published by the Free Software Foundation, either version 3 of the
-# License, or (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU Affero General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-"""
-OpenCraft - Views - Tests
-"""
-
-# Imports #####################################################################
-import ddt
-from django.conf import settings
-from django.test.utils import override_settings
-from django.urls import reverse
-
-from instance.tests.base import WithUserTestCase
-
-
-# Tests #######################################################################
-
-@ddt.ddt
-class IndexViewTestCase(WithUserTestCase):
-    """
-    Test cases for the default opencraft app view
-    """
-    url = reverse('index')
-    login_url = reverse(settings.LOGIN_URL)
-    instance_url = reverse('instance:index')
-    register_url = reverse('registration:register')
-    admin_url = reverse('admin:index')
-    admin_login_url = '{}?next={}'.format(reverse('admin:login'), admin_url)
-
-    def test_index_unauthenticated(self):
```

Comment:
Unauthenticated users should go to the login page.

Reply:
yeah, I have resolved all the reverse URL to `login` and `logout` accordingly :+1: